src/sdk/namespace_fs.js

```diff
@@ -447,7 +454,7 @@ class NamespaceFS {
         force_md5_etag,
     }) {
         dbg.log1('NamespaceFS: buffers_pool length',
-            buffers_pool.buffers.length, buffers_pool.sem);
+            multi_buffer_pool.buffer_pools);
```

Comment:
This prints undefined. We want to print the info of all the pools, so we best add a custom inspect method to buffer pool - see https://nodejs.org/dist/latest-v20.x/docs/api/util.html#custom-inspection-functions-on-objects